--- openh264-1.4.0/Makefile.orig	2015-03-05 10:25:12.000000000 +0100
+++ openh264-1.4.0/Makefile	2015-03-24 18:23:21.364389942 +0100
@@ -23,7 +23,7 @@
 SHARED=-shared
 OBJ=o
 DESTDIR=
-SHAREDLIB_DIR=$(PREFIX)/lib
+SHAREDLIB_DIR=$(LIBDIR)
 PROJECT_NAME=openh264
 MODULE_NAME=gmpopenh264
 GMP_API_BRANCH=Firefox38
@@ -239,12 +239,12 @@
 	install -m 644 codec/api/svc/codec*.h $(DESTDIR)/$(PREFIX)/include/wels
 
 install-static-lib: $(LIBPREFIX)$(PROJECT_NAME).$(LIBSUFFIX) install-headers
-	mkdir -p $(DESTDIR)/$(PREFIX)/lib
-	install -m 644 $(LIBPREFIX)$(PROJECT_NAME).$(LIBSUFFIX) $(DESTDIR)/$(PREFIX)/lib
+	mkdir -p $(DESTDIR)/$(LIBDIR)
+	install -m 644 $(LIBPREFIX)$(PROJECT_NAME).$(LIBSUFFIX) $(DESTDIR)/$(LIBDIR)
 
 install-static: install-static-lib $(PROJECT_NAME)-static.pc
-	mkdir -p $(DESTDIR)/$(PREFIX)/lib/pkgconfig
-	install -m 644 $(PROJECT_NAME)-static.pc $(DESTDIR)/$(PREFIX)/lib/pkgconfig/$(PROJECT_NAME).pc
+	mkdir -p $(DESTDIR)/$(LIBDIR)/pkgconfig
+	install -m 644 $(PROJECT_NAME)-static.pc $(DESTDIR)/$(LIBDIR)/pkgconfig/$(PROJECT_NAME)-static.pc
 
 install-shared: $(LIBPREFIX)$(PROJECT_NAME).$(SHAREDLIBSUFFIX) install-headers $(PROJECT_NAME).pc
 	mkdir -p $(DESTDIR)/$(SHAREDLIB_DIR)
@@ -252,10 +252,10 @@
 	if [ "$(SHAREDLIBSUFFIXVER)" != "$(SHAREDLIBSUFFIX)" ]; then \
 		cp -a $(LIBPREFIX)$(PROJECT_NAME).$(SHAREDLIBSUFFIX) $(DESTDIR)/$(SHAREDLIB_DIR); \
 	fi
-	mkdir -p $(DESTDIR)/$(PREFIX)/lib/pkgconfig
-	install -m 644 $(PROJECT_NAME).pc $(DESTDIR)/$(PREFIX)/lib/pkgconfig
+	mkdir -p $(DESTDIR)/$(LIBDIR)/pkgconfig
+	install -m 644 $(PROJECT_NAME).pc $(DESTDIR)/$(LIBDIR)/pkgconfig
 ifneq ($(EXTRA_LIBRARY),)
-	install -m 644 $(EXTRA_LIBRARY) $(DESTDIR)/$(PREFIX)/lib
+	install -m 644 $(EXTRA_LIBRARY) $(DESTDIR)/$(LIBDIR)
 endif
 
 install: install-static-lib install-shared
