--- openh264-2.4.1/build/platform-gnu-chain.mk.orig	2024-01-30 06:56:32.000000000 +0100
+++ openh264-2.4.1/build/platform-gnu-chain.mk	2024-02-28 18:05:31.669221886 +0100
@@ -12,9 +12,11 @@ STATIC_LDFLAGS += -lpthread -lm
 AR_OPTS = crD $@
 ifeq ($(ASM_ARCH), x86)
 ifeq ($(ARCH), x86_64)
+ifneq ($(IS_X32), Yes)
 ASMFLAGS += -f elf64
-else ifeq ($(ARCH), x32)
+else ($(ARCH), x32)
 ASMFLAGS += -f elfx32
+endif
 else
 ASMFLAGS += -f elf
 endif
--- openh264-1.4.0/codec/common/x86/asm_inc.asm~	2015-03-05 10:25:12.000000000 +0100
+++ openh264-1.4.0/codec/common/x86/asm_inc.asm	2015-04-26 11:58:16.516975386 +0200
@@ -122,6 +122,10 @@
 SECTION .note.GNU-stack noalloc noexec nowrite progbits ; Mark the stack as non-executable
 %endif
 
+%ifidn __OUTPUT_FORMAT__,elfx32
+SECTION .note.GNU-stack noalloc noexec nowrite progbits ; Mark the stack as non-executable
+%endif
+
 %define arg1 rdi
 %define arg2 rsi
 %define arg3 rdx
